.gitlab-ci.yml

```diff
@@ -0,0 +1,19 @@
+stages:
+  - build
+
+trigger_testbed:
+  stage: build
+  image:
+    name: registry.gitlab.sensirion.lokal/sensirion/docker/docker-ubuntu:18.04-1.0.0
+  tags:
+    - docker
+    - linux
+  script:
+  - >
+    curl
+    --request POST
+    --form token=$CI_JOB_TOKEN
```

Comment:
It's better to use the CI_JOB_TOKEN, since then GitLab can associate which project did trigger and create nice cross project pipeline graphs.

Reply:
Sure, makes sense.